packages/kolme/src/testtasks/kademlia_helper.rs

```diff
@@ -58,8 +19,11 @@ impl TestTasks {
     where
         F: FnOnce(GossipBuilder) -> GossipBuilder,
     {
-        let listener = GossipListener::random().unwrap();
-        let port = listener.port;
+        let port = std::net::TcpListener::bind("0.0.0.0:0")
```

Comment:
A small thought - if we're going to drop discovery mechanisms, it may be worth it to use some fixed TCP port to listen on? You mentioned at some point that we may need to workaround NATs, and having fixed port makes it a bit easier. And discovering other nodes in local network too.

Reply:
> it may be worth it to use some fixed TCP port to listen on

We still have the fixed TCP port from the websocket gossip manager.